.travis.yml

```diff
@@ -1,17 +1,20 @@
 language: python
 python:
-  - "2.7"
+  - 2.7
+  - 3.3
+  - 3.4
+  - 3.5
 sudo: false
 env:
-  - DJANGO_VERSION_CEILING=1.9.1 DJANGO_DB_ENGINE=sqlite
-  - DJANGO_VERSION_CEILING=1.9.1 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
-  - DJANGO_VERSION_CEILING=1.9.1 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
-  - DJANGO_VERSION_CEILING=1.10 DJANGO_DB_ENGINE=sqlite
-  - DJANGO_VERSION_CEILING=1.10 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
-  - DJANGO_VERSION_CEILING=1.10 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
+  - DJANGO_VERSION=1.8.10 DJANGO_DB_ENGINE=sqlite
+  - DJANGO_VERSION=1.8.10 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
+  - DJANGO_VERSION=1.8.10 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
+  - DJANGO_VERSION=1.9.3 DJANGO_DB_ENGINE=sqlite
```

Comment:
Can we leave these at 1.10 so we automatically test against new patch releases.  1.9.4 is already out.